test/integration/fedora-27/multiple.test

```diff
@@ -12,12 +17,20 @@ die() {
 }
 
 ko_to_test() {
-	tmp=${1%.ko}-LOADED.test
-	echo ${tmp#test-}
+	tmp=${1%${MODULE_POSTFIX}}${TEST_POSTFIX}
+	echo ${tmp#${MODULE_PREFIX}}
 }
 
 # make sure any modules added here are disjoint
-declare -a modules=(test-data-new.ko test-shadow-newpid.ko)
+declare -a modules
+
+for file in "${SCRIPTDIR}"/*"${TEST_POSTFIX}"; do
+	name=$(basename ${file})
+	if [ "${name}" == "${SCRIPTNAME}" ]; then
+		continue
```

Comment:
Since the postfix is -LOADED.test instead of .test, no need to check for the scriptname here.

Reply:
Right, removed this check in latest version.